geomesa-kafka/geomesa-kafka-datastore/pom.xml

```diff
@@ -98,4 +98,19 @@
         </dependency>
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>test-jar</goal>
```

Comment:
I'm somewhat tempted to suggest that the common 'test' bits into a sub-module so that one skipping tests can build.  (That's a separate issue.)